### PR TITLE
Auto OCP-29709

### PR DESCRIPTION
**Pull request by xiuwang**
@openshift/devexp-qe Please help review, thx

### CODE REVIEW

File: features/upgrade/samples_operator/upgrade.feature [hunk 1]
```diff
@@ -37,3 +37,99 @@ Feature: image-registry operator upgrade tests
...
+      | name        | pj-secret                                |
+      | from_file   | .dockerconfigjson=/tmp/.dockerconfigjson |
+      | type        | kubernetes.io/dockerconfigjson           |
+    Then the step should succeed
+    When I run the :import_image client command with:
+      | from             | registry.redhat.io/rhscl/ruby-25-rhel7:latest |
+      | confirm          | true                                          |
+      | image_name       | imagestream1:latest                           |
+      | reference-policy | local                                         |
+    Then the step should succeed
```
Comment (wewang58) [inline features/upgrade/samples_operator/upgrade.feature:67]: The project is include pull secret, still import failed?

Comment (xiuwang) [reply features/upgrade/samples_operator/upgrade.feature:67]: @wewang58 this is a upgrade prepare case on 4.4 which is not include the pull secret.

File: features/upgrade/samples_operator/upgrade.feature [hunk 2]
```diff
@@ -37,3 +37,99 @@ Feature: image-registry operator upgrade tests
...
+    And the output should not contain:
+      | error: Import failed |
+    When I run the :import_image client command with:
+      | from             | registry.redhat.io/rhscl/ruby-25-rhel7:latest |
+      | confirm          | true                                          |
+      | image_name       | imagestream2:latest                           |
+    Then the step should succeed
+    And the output should not contain:
+      | error: Import failed |
```
Comment (akostadinov) [inline features/upgrade/samples_operator/upgrade.feature:76]: Do we need string checks like this? They are unreliable check IMO and only make scenario longer.

I'd suggest to remove this and all other similar string checks unless you have seen actual value from them.

Comment (akostadinov) [reply features/upgrade/samples_operator/upgrade.feature:76]: other than that lgtm

Comment (xiuwang) [reply features/upgrade/samples_operator/upgrade.feature:76]: @akostadinov Import_image cmd return succeed whenever the image import succeed or fail. In result we need check if there are import failure to confirm the image import situation.

Comment (akostadinov) [reply features/upgrade/samples_operator/upgrade.feature:116]: @xiuwang , this deserves a bugzilla. A failed command should return a non-zero status. I'll merge but this is a bug.

**Pull request merged**